src/ember-app.js

```diff
@@ -188,6 +188,16 @@ class EmberApp {
     }
   }
 
+  /**
+   * Builds a new application instance sandbox as a micro-task.
+   */
+  buildNewApplicationInstance() {
```

Comment:
Should this function be marked as '@private'? What's the convention for using '_' for naming a method?